src/utils/logging_loss.jl

```diff
@@ -40,6 +40,16 @@ function lossfn(HM::LuxCore.AbstractLuxContainerLayer, x, (y_t, y_nan), ps, st,
     end
 end
 
+function lossfn(HM::Union{HybridModel, MultiNNHybridModel}, x, (y_t, y_nan), ps, st, logging::LoggingLoss)
```

Comment:
not sure that this union is the best way forward, maybe we could define an abstract type, e.g. `AbstractEasyHybrid` and then simply subtype on that for other models. We could refine this later in a follow up PR.